### PR TITLE
Allow RANCHER_SERVER to be set

### PR DESCRIPTION
**Pull request by adamgraves-choices**
RANCHER_URL was getting set before the RANCHER_SERVER variable was
pulled from Vault

### CODE REVIEW

File: common/answers/10-defaults.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@
 # SET SOME SANE DEFAULTS
 {
   echo "RACHER_SERVER=${RANCHER_SERVER-rancher.nhschoices.net}"
```
Comment (neilbmclaughlin) [inline common/answers/10-defaults.sh:5]: I'm not sure setting RACHER_SERVER  default is still needed. I could be wrong tho'!

File: common/deploy.sh [hunk 2]
```diff
@@ -22,10 +22,14 @@ check_rancher_vars
 
 echo "Deploying rancher stack $RANCHER_STACK_NAME in environment $RANCHER_ENVIRONMENT"
 
+
+# Set RANCHER_URL, doing it here so we can set RANCHER_SERVER in Vault
+echo "RANCHER_URL=https://${RANCHER_SERVER}/v2-beta/schemas"
```
Comment (neilbmclaughlin) [inline common/deploy.sh]: RANCHER_URL is checked for in check_rancher_vars(). Not sure if that is still a valid thing to do.

Comment (adamgraves-choices) [reply common/deploy.sh]: @neilbmclaughlin I've removed from the function now.

File: common/answers/10-defaults.sh (hunk 1 above)
Comment (adamgraves-choices) [reply common/answers/10-defaults.sh:5]: @neilbmclaughlin you're right, something to remove at some point in the future.

**Review by neilamoran — APPROVED** [review]

**Pull request merged**